### ISSUE TITLE
[REMOTO] [Natal - RN] Engenheiro de Deep Learning na Deeper Systems 

### ISSUE THREAD

**Issue by joannaml** [minas-ti/vagas#142]
## Descrição da vaga

A DS oferece soluções revolucionárias para problemas de negócios usando visão computacional, aprendizado de máquina e inteligência artificial. Adotamos abordagens inovadoras para identificar e eliminar as ineficiências corporativas. Através do fluxo de trabalho e da análise de dados para o desenvolvimento do algoritmo, a Deeper Systems implementa inovações tecnológicas em todos os aspectos.

O objetivo da função é extrair cada estágio do desempenho dos códigos de aprendizagem profunda por meio da aplicação de criatividade teórica e intuição técnica. Você estará contribuindo para a ruptura das soluções tradicionais ao desenvolver algoritmos.

## Local

Remoto

#### Diferenciais

- salário fixo mensal;
- horário flexível.

## Requisitos

**Obrigatórios:**
- fortes habilidades em Python enquanto trabalhava com TensorFlow e Keras;
- conhecimento prático e teórico sobre aprendizado de máquina e redes neurais;
- experiência com processamento de imagens e redes convolucionais.

**Diferenciais:**
- experiência com PyTorch;
- conhecimento de computação GPU (CUDA, OpenCL, OpenACC) e HPC (MPI, OpenMP);
- conhecimento dos componentes internos do compilador desde o front-end até o ambiente de execução;
- experiência prática com big data;
- familiaridade com técnicas de big data como Hadoop, Spark, Cassandra;
- grande capacidade de comunicação e organização.

## Contratação

PJ

## Nossa empresa

A DS oferece soluções revolucionárias para problemas de negócios usando visão computacional, aprendizado de máquina e inteligência artificial. Adotamos abordagens inovadoras para identificar e eliminar as ineficiências corporativas. Através do fluxo de trabalho e da análise de dados para o desenvolvimento do algoritmo, a Deeper Systems implementa inovações tecnológicas em todos os aspectos.

Encorajamos mentes receptivas a contribuir para o objetivo mais amplo por meio do desenvolvimento de seu potencial individual. O candidato ideal tem uma atitude fantástica e diligente e se sente à vontade para enfrentar qualquer obstáculo. 

Oferecemos um ambiente criativo e fácil para indivíduos que trabalham arduamente. Se você tem espírito empreendedor e está ansioso para sujar as mãos, mergulhe de cabeça e veja como funciona a verdadeira ciência!

## Como se candidatar

Se você tem interesse em falar conosco,preencha o formulário de inscrição: https://cutt.ly/PlxWo85

## Labels

- 🏢 Flexível
- 🏢 Remoto
- 👶 Estágio
- 👦 Júnior
- 👨 Pleno
- 👴 Sênior
- ⚖️ PJ
